src/ec-evaluator/types.ts

```diff
@@ -90,3 +93,20 @@ export type CmdEvaluator = (
 // Special value that cannot be found on the stash so is safe to be used
 // as an indicator of an error from running the ECE machine
 export class ECError {}
+
+// Types that can be stored in env and assigned to variables
+export type StorageType = es.Literal | List | ClosureInstr
```

Comment:
Take note: these are the only types that should be stored in environment (and returned at the end of evaluation). 

In order for us to format the output with the type, all "StorageType"s should not be pure literals/lists/etc., but wrapped with their types.